pkg/datasource/schedule.go

```diff
@@ -44,10 +46,10 @@ func (datasource *MsupplyEresDatasource) GetSchedules() ([]Schedule, error) {
 	defer rows.Close()
 
 	for rows.Next() {
-		var ID, Name, Description, ReportGroupID, Time, Lookback string
+		var ID, Name, Description, ReportGroupID, Time, Lookback, DateFormat, DatePosition string
 		var Day, Interval, NextReportTime int
 
-		err = rows.Scan(&ID, &Interval, &NextReportTime, &Name, &Description, &Lookback, &ReportGroupID, &Time, &Day)
+		err = rows.Scan(&ID, &Interval, &NextReportTime, &Name, &Description, &Lookback, &ReportGroupID, &Time, &Day, &DateFormat)
```

Comment:
I am getting an error saying 
```
 Could not scan schedule rows: excel-report-email-scheduler/pkg/datasource.(*MsupplyEresDatasource).GetSchedules: sql: expected 11 destination arguments in Scan, not 9"
```

Means you are missing fields here that are supposed to matched to a variable.